website/views/organization.py

```
@@ -3930,3 +3955,189 @@ def extract_payment_info_from_comments(self, issue, owner, repo_name, issue_numb
+            messages.error(request, "Received invalid data from Slack.")
+
+    if request.method == "POST":
+        selected_channel = request.POST.get("target_channel")
+        welcome_message = request.POST.get("welcome_message", "").strip()
+        daily_updates = request.POST.get("daily_sizzle_timelogs_status") == "on"
+
+        hour_value = request.POST.get("daily_sizzle_timelogs_hour")
+        try:
+            hour_value = int(hour_value) if hour_value else None
+            if hour_value is not None and not (0 <= hour_value <= 23):
+                raise ValueError()
```

Ruff (0.14.5)

Abstract `raise` to an inner function

(TRY301)